pkg/asset/installconfig/powervs/metadata.go

```diff
@@ -84,7 +94,7 @@ func (m *Metadata) APIKey(ctx context.Context) (string, error) {
 }
 
 // GetCISInstanceCRN gets the CRN name for the specified base domain.
-func GetCISInstanceCRN(APIKey string, BaseDomain string) (string, error) {
+func (m *Metadata) GetCISInstanceCRN() (string, error) {
```

Comment:
While you're changing things, can you change this function name to something like CISCRNName(), or something? None of the other "getters" have `Get` in the name, and there are two the retrieve CRN-related values, so I think this would make the purpose of each more clear. Unless this ends up creating a cascade of other things that need changing and making this PR overly-large.